packages/runtime/src/utils/esm-compat.ts

```diff
@@ -0,0 +1,2 @@
+// see https://www.npmjs.com/package/vite-plugin-cjs-interop
+export const esmCompat = <T>(x: T): T => (x as any).default ?? x
```

Comment:
Factored this out in a separate file for clarity/easier search of all places where we apply it.